### ISSUE TITLE
"Structure element" -> "Footprint"

### ISSUE THREAD

**Issue by haesleinhuepf** [BiAPoL/Quantitative_Bio_Image_Analysis_with_Python_2022#31]
low priority, just in case there is time: The scikit-image developers are subsequently renaming "Strutcutal element" to "Footprint". We may want to adapt that in our materials.

See also:
https://github.com/scikit-image/scikit-image/pull/5445

**Comment by zoccoler** [comment]
Hi Robert @haesleinhuepf ,

I would say that we can swap focus and let "footprint" be the main term because we show mostly scikit-image. By the way, I like this standardization. However, I would not entirely remove "structural element" because much of the literature about morphological operations refers to this term. To my knowledge, footprint is somewhat specific to scikit-image. For example, MATLAB uses "SE" (https://www.mathworks.com/help/images/ref/imdilate.html).

**Comment by haesleinhuepf** [comment]
Yes, fully agreed. I also propose to keep the content about SEs minimal because it's not so relevant for our work.

**Comment by haesleinhuepf** [comment]
Btw `SE` as variable name violates good practice in Python 😉
https://peps.python.org/pep-0008/#function-and-variable-names

**Comment by zoccoler** [comment]
> Btw `SE` as variable name violates good practice in Python 😉 https://peps.python.org/pep-0008/#function-and-variable-names

Totally! Luckily this one is not in Python 😆
